browser/src/core/Socket.js

```diff
@@ -1476,6 +1476,9 @@ app.definitions.Socket = L.Class.extend({
 
 	_onSocketClose: function () {
 		console.debug('_onSocketClose:');
+		if (this.ReconnectCount > 0)
+			return;
+
 		var isActive = this._map._active;
 		this._map.hideBusy();
```

Comment:
Btw, why aren't we only skipping the hideBusy for this fix, instead of returning early?

For example:
```
if (this.ReconnectCount <= 0) {
    this._map.hideBusy();
}
```
And leave all the other logic as-is. Wouldn't that be safer? We only want to show the busy label when reconnecting, no? If not, then what are we trying to do?

Reply:
Because if you have test your proposal, then you will show an undesired side effects with vex open message dialogs,
not related to reconnecting.

>> We only want to show the busy label when reconnecting, no? If not, then what are we trying to do?
Yes, it only changes the label, no more further changes. and if you analyze:

t1: re-connection starts 
t2: websocket close because no success connection
t3: _onSocketClose executes with all undesired side effects like vex open dialogs messages
t4: re-connection starts again Count+1

If you test without these changes, you will notice that 2 vex dialogs opens, in sequence

Reply:
> What happens when reconnection doesn't work (but the document was loaded)? (This will happen when the server crashes.)
_onSocketClose will be called

> What happens when loading the document doesn't work, and we have to retry? (Do we give up now? Do we give up after retrying a few times?)
re-connection start  and until  Max Count retries, after that it will give up with a polite vex message dialog

> What happens when the server is shut down (cleanly) after loading a doc, and then started up, after a minute or two? (This is a supported "recycling" scenario.)

on Client Side, what ever reason if the link websocket is disconnected, it will execute _onSocketClose

Reply:
> If the above scenarios work correctly, then we should be good. Please confirm they work after trying multiple times, to be sure.

The scenarios you describe is correct, but I just only changing instead to show a vex dialog to a label message,
there is a test file case, I will link you to test it, I can confirm it works